app/views/admin/accounting/transactions/_modal_content.slim

```diff
@@ -0,0 +1,22 @@
+div.modal-header
+  button type="button" class="close cancel" data-dismiss="modal" aria-label== t(:cancel)
+    span aria-hidden="true" &times;
+  h4.modal-title Add Transaction
+
+div.modal-body
+  = render 'alerts'
+
+  = simple_form_for [:admin, @transaction], remote: true do |f|
+    = error_notification(f)
+
+    = f.hidden_field :project_id
+
+    = f.association :account
+
+    = f.input :amount
+
+    = f.input :private_note
```

Comment:
Wow, much cleaner!!! Can you ditch the blank lines between the fields? I don't think they're necessary.